practice_2/cells/CustomTableViewCell1.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+class CustomTableViewCell1: UITableViewCell {
```

Comment:
make more informative names

Reply:
Hello, thanks.

I thought a lot about how to name this class. I have heard that good practice is to repeat the name of the type in the name. (e.g. if a Label, then we call the variable as  "mainLabel"). Is this true? On the other hand I heard that "using names based on roles, not types" (source: https://github.com/raywenderlich/swift-style-guide)

As for "CustomTableViewCell1" I agree that the name is too abstract. Maybe "TopTableViewCell"? But in general, this cell can be used not only at the top of storyboard. Please advise, what is the most correct name for this class? Thanks in advance

Reply:
it's up to you
some suggestions
HorizontalTableViewCell
SquareTableViewCell
ViewPagerTableViewCell

https://swift.org/documentation/api-design-guidelines/
https://betterprogramming.pub/naming-conventions-in-swift-4b7ca5eed4d2

Reply:
Thanks. Done